demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -930,3 +933,113 @@ def test_IsDeprecatedCorrectlyValidator_is_valid():
     for result in results:
         assert result.content_object.deprecated
         assert result.content_object.description == "Some description"
+
+
+@pytest.mark.parametrize(
+    "content_items",
+    [
+        pytest.param(
+            [
+                create_integration_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case integration with trailing spaces in name",
+        ),
+        pytest.param(
+            [
+                create_classifier_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case classifier with trailing spaces in name",
+        ),
+        pytest.param(
+            [
+                create_dashboard_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case dashboard with trailing spaces in name",
+        ),
+        pytest.param(
+            [
+                create_incident_type_object(
+                    paths=["name"], values=["name_with_space_should_fail "]
+                )
+            ],
+            id="case incident type with trailing spaces in name",
+        ),
+        pytest.param(
+            [create_wizard_object({"name": "name_with_space_should_fail "})],
+            id="case wizard with trailing spaces in name",
+        ),
+    ],
+)
+def test_IsContentItemNameContainTrailingSpacesValidator_is_valid(content_items):
+    """Test validate BA113 - Trailing spaces in content item name
+    Given:
+        A list of content items with names that have trailing spaces.
```

Comment:
we pass one every time, can create the list inside the test, then no need for the `all`